src/lib/eina/eina_inline_private.h

```diff
@@ -20,8 +20,9 @@
 # define EINA_INLINE_PRIVATE_H_
 
 # include <time.h>
+
 #ifndef _MSC_VER
-# include <sys/time.h>
+# include <evil_private.h>
```

Comment:
this should be at the `else` part, as `evil_private` **should** be included when on a native-windows build.